lib/modules/apostrophe-utils/lib/api.js

```diff
@@ -616,13 +616,38 @@ module.exports = function(self, options) {
 
   // Identical to `apos.utils.warn`, except that the warning is
   // not displayed if `process.env.NODE_ENV` is `production`.
+  // Also see `warnDevOnce` which is less likely to irritate
+  // the developer until they stop paying attention.
+
   self.warnDev = function(msg) {
     if (process.env.NODE_ENV === 'production') {
       return;
     }
     self.warn.apply(self, arguments);
   };
 
+  self.warnedDev = {};
```

Comment:
I was confused here thinking you were overriding `warnDev`. Optional suggested change is to make this something like `sentWarning` or `devWarned` for clarity and decreased human error.